scripts/kc_data_transfer/src/KeyCloakTransfer.py

```diff
@@ -117,9 +119,7 @@ def __init__(self):
         #   FOM_SUBMITTER_TESTING_00001011, FOM_SUBMITTER_TESTING_00001012... etc
         self.fom_submitter_parent_role_name = "FOM_SUBMITTER"
         self.fom_reviewer_role_name = "FOM_REVIEWER"
-        self.fam = FamWrapper(
-            fom_forest_client_role_parent=self.fom_submitter_parent_role_name
```

Comment:
the fom_forest_client_role_parent parameter is not in use at all, just removed it